pkg/controllers/clusterimport/manager.go

```diff
@@ -15,39 +14,11 @@ import (
 
 	"sigs.k8s.io/controller-runtime/pkg/controller"
 	"sigs.k8s.io/controller-runtime/pkg/event"
-	"sigs.k8s.io/controller-runtime/pkg/handler"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 	"sigs.k8s.io/controller-runtime/pkg/predicate"
-	runtimesource "sigs.k8s.io/controller-runtime/pkg/source"
 )
 
-const controllerName = "autoimport-controller"
-
-func SetupCSRApprovalController(mgr manager.Manager, kubeClient kubernetes.Interface) error {
```

Comment:
no CSR required?

Reply:
we don't need need this any more, the ocm controlplane agent will handle this